src/components/Settings.tsx

```diff
@@ -212,7 +239,7 @@ export default function Settings() {
       {<select {...register3('currentLearnLanguageId')} defaultValue={user?.learnLanguageId} className="input appearance-none rounded-none w-2/3 px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-b-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm">
       {languages.map((lang) => <option key={lang.id} value={lang.id}>{lang.name} </option>)}
```

Comment:
You could capitalize the language names (use `capitalize` function from the navbar) and also include the flag (`lang.flag`).

Reply:
> 

Will add those!